subsys/settings/src/settings_fcb.c

```diff
@@ -13,6 +13,18 @@
 #include "settings/settings_fcb.h"
 #include "settings_priv.h"
 
+#if USE_PARTITION_MANAGER
```

Comment:
just why this is not incorporated into a header file?

Reply:
It should be.

I'll add that to the partition manager TODO list.

Reply:
:D